webapp/src/components/checklist_item.tsx

```diff
@@ -511,14 +513,16 @@ export const ChecklistItemDetails = (props: ChecklistItemDetailsProps): React.Re
 };
 
 interface ChecklistItemEditModalProps {
-    show: boolean
-    onDone: () => void
-    checklistNum: number
-    playbookRunId: string
-    itemNum: number
-    taskTitle: string
-    taskDescription: string
-    taskCommand: string
+    show: boolean;
+    onDone: () => void;
+    checklistNum: number;
+    playbookRunId: string;
+
+    channelId: string;
+    itemNum: number;
+    taskTitle: string;
+    taskDescription: string;
+    taskCommand: string;
```

Comment:
Should we remove the blank line?
```suggestion
    show: boolean;
    onDone: () => void;
    checklistNum: number;
    playbookRunId: string;
    channelId: string;
    itemNum: number;
    taskTitle: string;
    taskDescription: string;
    taskCommand: string;
```